website/docs/commands/cli-config.html.markdown

```diff
@@ -63,35 +63,89 @@ The following settings can be set in the CLI configuration file:
   [plugin caching](/docs/configuration/providers.html#provider-plugin-cache)
   and specifies, as a string, the location of the plugin cache directory.
 
-- `credentials` — provides credentials for use with Terraform Cloud.
-    Terraform uses this when performing remote operations or state access with
-    the [remote backend](../backends/types/remote.html) and when accessing
-    Terraform Cloud's [private module registry.](/docs/cloud/registry/index.html)
-
-    This setting is a repeatable block, where the block label is a hostname
-    (either `app.terraform.io` or the hostname of a Terraform Enterprise instance) and
-    the block body contains a `token` attribute. Whenever Terraform accesses
-    state, modules, or remote operations from that hostname, it will
-    authenticate with that API token.
-
-    ``` hcl
-    credentials "app.terraform.io" {
-      token = "xxxxxx.atlasv1.zzzzzzzzzzzzz"
-    }
-    ```
-
-    ~> **Important:** The token provided here must be a
-    [user token](/docs/cloud/users-teams-organizations/users.html#api-tokens)
-    or a
-    [team token](/docs/cloud/users-teams-organizations/api-tokens.html#team-api-tokens);
-    organization tokens cannot be used for command-line Terraform actions.
-
-    -> **Note:** The credentials hostname must match the hostname in your module
-    sources and/or backend configuration. If your Terraform Enterprise instance
-    is available at multiple hostnames, use one of them consistently. (The SaaS
-    version of Terraform Cloud responds to API calls at both its current
-    hostname, app.terraform.io, and its historical hostname,
-    atlas.hashicorp.com.)
+- `credentials` - configures credentials for use with Terraform Cloud, Terraform
+Enterprise, or other hosts that provide Terraform services. See
```

Comment:
> or other hosts that provide Terraform services
Does this refer to a specific thing, or potential third-party implementations? If the later, I wouldn't mention it here (we don't with the registry), and instead save that information for the `internals` documentation.

Reply:
As far as I'm aware it's referring to potential third-party implementations of "Terraform services", i.e. the remote backend or registry. Agreed that it would be clearer to omit here, so I've made that change.

I'm not sure where in the internals docs we could add this information—it doesn't seem to fit in the docs for credentials helpers, and we don't have other internal docs for credentials. If you have suggestions I'd be happy to add it back, but I think it's also fine to leave it implicit.

Reply:
👍 sounds good to me!